app/modules/toolkits/cdk_wrapper.py

```diff
@@ -394,6 +398,7 @@ def get_tanimoto_similarity_CDK(
     mol2: any,
     fingerprinter: str = "PubChem",
     ECFP: int = 6,
```

Comment:
For get_tanimoto_similarity_ECFP_CDK() you're using ECFP2 as default, here it's ECFP6. I would think about using the same default for everything.